kw.sh

```diff
@@ -46,35 +90,60 @@ function kw()
       (
         . $src_script_path/vm.sh --source-only
 
+        check_alert_option $@
+        local alert=$?
         vm_prepare
+        local ret=$?
+        alert_completion $alert "kw prepare"
```

Comment:
Here, after add paplay or any other player, I recommend you to use:

```sh
alert_completition $alert "kw prepare" &
```
with this we detach the execution of the sound, and make things a little bit more "natural".

Reply:
Nice! I think it would be better to add the "&" in paplay execution (inside alert_completion function), in order to detach sound execution, directly, don't you think?